load-tests/user-files/simulations/NtbsLoadTest.scala

```diff
@@ -0,0 +1,108 @@
+import scala.concurrent.duration._
+import scala.util.Random
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.jdbc.Predef._
+
+class NtbsLoadTest extends Simulation {
+
+    val httpProtocol = http
+        .baseUrl("https://ntbs-load-test.e32846b1ddf0432eb63f.northeurope.aksapp.io")
+        .inferHtmlResources(WhiteList("""https://ntbs-load-test.e32846b1ddf0432eb63f.northeurope.aksapp.io/.*"""), BlackList())
```

Comment:
Is it worth pulling these out as config parameters?

Reply:
Yes, I think this is a good idea